@xen-orchestra/lite/src/components/AppLogin.vue

```diff
@@ -13,13 +13,10 @@
           v-model="password"
           :placeholder="$t('password')"
           :readonly="isConnecting"
+          required
         />
       </FormInputWrapper>
-      <UiButton
-        type="submit"
-        :busy="isConnecting"
-        :disabled="password.trim().length < 1"
-      >
+      <UiButton type="submit" :busy="isConnecting">
```

Comment:
IMO this is not the right way to solve the problem.
If the problem is that the contrast is not enough, the problem should be solved by updating the colors.
This need to be talked with Clemence.

Reply:
> IMO this is not the right way to solve the problem. If the problem is that the contrast is not enough, the problem should be solved by updating the colors. This need to be talked with Clemence.

IMHO, we don't need to disable the login button. Therefore, setting the input as required is the appropriate behavior that we need in this case.